app/src/main/java/io/left/meshim/adapters/ConversationListAdapter.java

```diff
@@ -75,7 +76,17 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent)
             userName.setText(conversationSummary.username);
             TextView newMessage = v.findViewById(R.id.userNewestMessageText);
             newMessage.setText(conversationSummary.messageText);
-            Date currDate = conversationSummary.messageTime;
+            TextView newMessageBadge = v.findViewById(R.id.newMessageNotificationBadge);
+            if (!conversationSummary.isRead) {
+                //show how many unread messages there are.
+                newMessageBadge.setVisibility(View.VISIBLE);
+                newMessage.setTypeface(null, Typeface.BOLD);
+                newMessage.setTextColor(Color.BLACK);
```

Comment:
Are these not defined in XML?

Reply:
No so I wanted to show unread messages in Bold so that they would stand out from the read messages in the conversation list. This was before we added the number of unread messages badge.

Reply:
It looks neater if unread messages were bold